templates/ts/custom-copilot-rag-azure-ai-search/src/app/turnState.ts

```diff
@@ -0,0 +1,12 @@
+import { DefaultConversationState, TurnState } from "@microsoft/teams-ai";
+
+export interface Task {
```

Comment:
Seems not used, right? If so, can delete this file and use the default conversation state.